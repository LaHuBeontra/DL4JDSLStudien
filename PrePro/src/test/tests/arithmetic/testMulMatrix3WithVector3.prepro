function main() {
	import mat3 testMat3_1, vec3 p1;

	vec3 result = testMat3_1 * p1;

	export result;
}
