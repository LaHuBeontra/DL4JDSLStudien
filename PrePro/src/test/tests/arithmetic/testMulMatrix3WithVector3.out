DefaultPreProDataSet{
dataSetEntries=
[DataSetEntry{variableName='result', variableShape=[4, 3], startColumn=0, length=3}],
dataSet=
[[42.00,  168.00,  294.00],  
 [10.00,  13.00,  16.00],  
 [95.00,  110.00,  125.00],  
 [168.00,  186.00,  204.00]]

constants={}

calculated_variables=
result=[[42.00,  168.00,  294.00],  
 [10.00,  13.00,  16.00],  
 [95.00,  110.00,  125.00],  
 [168.00,  186.00,  204.00]]
}